auth_backends/__init__.py

```diff
@@ -3,4 +3,4 @@
  These package is designed to be used primarily with Open edX Django projects, but should be compatible with non-edX
  projects as well.
 """
-__version__ = '2.0.0'  # pragma: no cover
+__version__ = '2.1.0'  # pragma: no cover
```

Comment:
I would say bump to 3.0.0, since we are dropping supported platforms (ones that likely don't matter to anyone at this point, but still).

Reply:
@mikix I've rebased this and changed the version to `3.0.0` as you suggested. Do you know if there's anyone who could help with fixing `test_backends.py` to work with the changes to the upstream OpenIdConnectTestMixin ?

Reply:
I tried to look at fixing them, and it was more complicated than I had time for. I'm trying to find someone at edX that would know more.

Reply:
> I tried to look at fixing them, and it was more complicated than I had time for. 

Same here, haha. Thanks!